app/views/candidate_interface/apply_from_find/not_found.html.erb

```diff
@@ -3,12 +3,12 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-xl">
-      <%= service_name %>
+      There is a problem
     </h1>
 
     <p class="govuk-body">
-      We could not find the course you’re looking for – there may be a problem with the training provider and/or course code(s).
-      You can go back to <%= govuk_link_to 'Find postgraduate teacher training', I18n.t('find_postgraduate_teacher_training.production_url') %> and try selecting your course again, or choose a different course.
+      We could not find the course you’re looking for. There may be a problem with the training provider or the course.
+      <%= govuk_link_to 'Try selecting the course again', I18n.t('find_postgraduate_teacher_training.production_url') %> or find a different course to apply to.
```

Comment:
Could this whole sentence be a link rather ? Excluding the full stop at the end of course!